src/Infrastructure/Persistence/Configuration/CommunityConfiguration.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Codidact.Domain.Entities;
+using Codidact.Domain.Enums;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Codidact.Infrastructure.Persistence.Configuration
+{
+    public class CommunityConfiguration : IEntityTypeConfiguration<Community>
+    {
+        public void Configure(EntityTypeBuilder<Community> builder)
+        {
+            builder.Property(t => t.Name)
+                .HasMaxLength(40)
+                .IsRequired();
+
+            builder.HasIndex(t => t.Name)
+                .IsUnique(true);
+
+            builder.Property(t => t.Tagline)
+                .HasMaxLength(100)
+                .IsRequired();
+
+            builder.HasIndex(t => t.Tagline)
+               .IsUnique(true);
```

Comment:
Question: do taglines need to be unique, as long as community names are? (Granted they probably _will_ be, but do we need to _enforce_ it?)

Reply:
I just followed the forum but I agree.

Reply:
Fixed f180c9630f9b19de9d92a5ec42050dac09161ca2